packs/caddy/templates/caddy.nomad.tpl

```diff
@@ -0,0 +1,85 @@
+job [[ template "job_name" . ]] {
+[[ template "region" . ]]
+datacenters = [ [[ range $idx, $dc := .caddy.datacenters ]][[if $idx]],[[end]][[ $dc | quote ]][[ end ]] ]
+  namespace = [[ .caddy.namespace | quote ]]
+
+  type = "system"
```

Comment:
Can we make this a variable? Another alternative is to run these types of job a service jobs with a known count.